projects/js-packages/components/components/status/index.tsx

```diff
@@ -0,0 +1,41 @@
+import { __ } from '@wordpress/i18n';
+import classNames from 'classnames';
+import styles from './style.module.scss';
+
+interface StatusProps {
+	status?: 'active' | 'error' | 'inactive' | 'action' | 'initializing';
+	label?: string;
+	className?: string;
+}
+
+const Status = ( { className, label, status = 'inactive', ...rest }: StatusProps ): JSX.Element => {
```

Comment:
Agree with @retrofox that we should not use rest props since it opens for warnings and problems, and the reason exists an eslint rule for it (that we should use maybe): https://github.com/jsx-eslint/eslint-plugin-react/blob/master/docs/rules/jsx-props-no-spreading.md

Reply:
Fixed! https://github.com/Automattic/jetpack/pull/30209/commits/097cca19811ecab4023604347f2f7136930bf731 :+1: